easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.2.2-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -92,6 +93,7 @@ components = [
             'dfda4dd5f9aba19fe2b6eb9a0ec583d12dcefdfee8ab8803fc57ad48d582db04',  # AlphaFold-2.0.0_n-cpu.patch
             '5363d403baf5ab73f4d3ddd72e19af9ff832de4b1d7ba25a5fbcc5846c1c890f',  # AlphaFold-2.1.0_fix-scp-path.patch
             '1a2e4e843bd9a4d15ee39e6c37cc63ba281311cc7a0a5610f0e43b52ef93faac',  # AlphaFold-2.0.1_setup_rm_tfcpu.patch
+            'fb00a3c9cb00fe87eef77d48dfd8e44c40ad8b820480c79c5937377da27fea5f',  # AlphaFold-2.2.2_jax038_NaN.patch
```

Comment:
```suggestion
            '0b040ba99208cd9ad79b7d17db5720f7a447a019ac0ca786b3b019a55d1544e0',  # AlphaFold-2.2.2_jax038_NaN.patch
```

Reply:
Oh, I overlooked that `--inject-checksums` doesn't handle component checksums... 🤦 

Fixed in fc8be7736b (which was force pushed into the PR)